tests/experimental/metric_repository/test_column_descriptive_metrics_metric_retriever_integration.py

```diff
@@ -64,4 +65,38 @@ def test_get_metrics(
             value=["col1", "col2"],
             exception=None,
         ),
+        # TableMetric[List[Dict[str, str]]](
+        #     batch_id=batch_id,
+        #     metric_name="table.column_types",
+        #     value=[{'name': 'col1', 'type': int}, {'name': 'col2', 'type': int}],
+        #     exception=None,
+        # ),
+        ColumnMetric[int](
```

Comment:
This was strange : I run the test and the lists are identical, but the assertion fails. Maybe it's something simple I'm missing?